config/clusters/leap/common.values.yaml

```diff
@@ -177,7 +176,12 @@ basehub:
                   slug: "pytorch"
                   kubespawner_override:
                     image: "pangeo/pytorch-notebook:2024.04.08"
-          kubespawner_override: &medium_kubespawner_override
+                leap-pangeo-edu:
+                  display_name: LEAP Education Notebook (Testing Prototype)
+                  slug: "leap_edu"
+                  kubespawner_override:
+                    image: "quay.io/jbusecke/leap-edu-image:fa442ab4851c"
+          kubespawner_override:
```

Comment:
This addition allows us to not duplicate this list of images - which if we do causes a need to update the same image tags in multiple profiles